modules/javafx.fxml/src/main/java/javafx/fxml/FXMLLoader.java

```diff
@@ -1497,6 +1490,11 @@ public void processCharacters() throws IOException {
 
             updateValue(text.trim());
         }
+
+        @Override
+        public void processEndElement() {
+            // No-op
+        }
```

Comment:
Why is this method added?

Reply:
I'd recommend reverting the changes around process*() methods.